sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -4530,6 +4560,11 @@ enqueueMemFillHelper(pi_command_type CommandType, pi_queue Queue, void *Ptr,
                                                          &ZeFence))
     return Res;
 
+  _pi_ze_event_list_t TmpWaitList;
```

Comment:
Since you reverted passing the command-list please undo the move of this code too ? (and in other functions too)

Reply:
sure, will do.